VERSION.h

```diff
@@ -0,0 +1,6 @@
+// THIS FILE IS AUTOMATICALLY GENERATED
+// Run `make VERSION.h` to update it after modifying VERSION.
+unsigned char VERSION[] = {
+  0x30, 0x2e, 0x34, 0x2e, 0x30, 0x0a
```

Comment:
:(
I hate committing build artifacts but I don't know of a better way to handle this. The alternative is duplicate the version into the top of the C file. But then we need to maintain versions in sync in README/setup.py/dumb-init.c/debian changelog.